taxi_service/settings.py

```diff
@@ -124,6 +124,8 @@
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [BASE_DIR / "static"]
```

Comment:
Ensure that the file ends with a single blank line, as required by the checklist.